lib/ansible/parsing/mod_args.py

```diff
@@ -115,6 +115,10 @@ def __init__(self, task_ds=None, collection_list=None):
             raise AnsibleAssertionError("the type of 'task_ds' should be a dict, but is a %s" % type(task_ds))
         self._task_ds = task_ds
         self._collection_list = collection_list
+        from ansible.playbook.handler import Handler  # delayed local import to prevent circular import
```

Comment:
I assume that we are using Handler here because it is a superset of all attrs a task could use?  Might be a good idea to add a short comment.

Reply:
Yeah, I originally had it merging the sets from handler/task... Might be good to go back to that, just in case they ever diverged (plus it's a little more obvious *why* that way)